app/waf/src/controllers/adc.controller.ts

```diff
@@ -458,12 +476,15 @@ export class AdcController {
       this.reqCxt.get(WafBindingKeys.Request.KeyUserToken),
       this.reqCxt.get(WafBindingKeys.Request.KeyTenantId),
     ]).then(async ([computeHelper, userToken, tenantId]) => {
-      let rootPass = Math.random()
-        .toString(36)
-        .slice(-8);
-      let adminPass = Math.random()
-        .toString(36)
-        .slice(-8);
+      // TODO: uncomment me.
```

Comment:
Temporarily comment on the password generation since we have lots of manual tests which needs the password inputting, restore them to 'default' and 'admin' would be easy for current developments.